src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindSVBreakpointsSparkUnitTest.java

```diff
@@ -0,0 +1,166 @@
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMUtils;
+import htsjdk.samtools.fastq.FastqReader;
+import htsjdk.samtools.fastq.FastqRecord;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceWindowFunctions;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.BaseUtils;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import scala.Tuple2;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Unit tests for FindSVBreakpointsSpark.
+ */
+public class FindSVBreakpointsSparkUnitTest extends BaseTest {
+
+    private static final String REFERENCE_FILE_NAME = hg19MiniReference;
+    private static final List<GATKRead> TEST_READS;
+    static {
+        TEST_READS = new ArrayList<>(4);
+        final SAMFileHeader samHeader = ArtificialReadUtils.createArtificialSamHeader();
+        TEST_READS.addAll(ArtificialReadUtils.createPair(samHeader, "firstReadPair", 101, 1010, 1382, false, false));
+        TEST_READS.addAll(ArtificialReadUtils.createPair(samHeader, "secondReadPair", 101, 1210, 1582, false, false));
+        for ( int idx = 0; idx != 4; ++idx ) {
+            final GATKRead read = TEST_READS.get(idx);
+            read.getBases()[0] = BaseUtils.BASES[idx];
+            read.getBaseQualities()[0] = (byte)(30 + idx);
+            read.setMappingQuality(FindSVBreakpointsSpark.BreakpointClusterer.MIN_MAPQ);
+        }
+        TEST_READS.get(0).setCigar("51S50M");
+        TEST_READS.get(2).setIsProperlyPaired(false);
+        TEST_READS.get(2).setMateIsReverseStrand(false);
+        TEST_READS.get(3).setIsProperlyPaired(false);
+        TEST_READS.get(3).setIsReverseStrand(false);
+    }
+    @Test(groups = "spark")
+    void kmersToIgnoreTest() throws IOException {
+        final JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
+        final ReferenceMultiSource ref = new ReferenceMultiSource((PipelineOptions)null,
+                REFERENCE_FILE_NAME,
+                ReferenceWindowFunctions.IDENTITY_FUNCTION);
+        final List<SVKmer> badKmers = FindBadGenomicKmersSpark.findBadGenomicKmers(ctx, ref, null, null);
+        final File kmersFile = File.createTempFile("kmerKillList", "txt");
+        kmersFile.deleteOnExit();
+        FindBadGenomicKmersSpark.writeKmersToFile(kmersFile, badKmers);
+        final Set<SVKmer> kmerSet = FindSVBreakpointsSpark.readKmersToIgnore(kmersFile);
+        if ( !kmersFile.delete() )
```

Comment:
no need for this checking for deletions unless that's actually part of the what you're testing